### ISSUE TITLE
Chapter 2 

### ISSUE THREAD

**Issue by jingzhi93** [CertifaiAI/Intro_to_Pytorch#3]
Add the title

"How to perform Autograd"

-  try to use tensor.requires_grad properties to show the availability of grad

2.2 linear regression example

-  change the font of X,Y and w to math form

"Linear Regression using PyTorch with Autograd"

- a bit confusing with the statement "With autograd, we no need to manually calculate gradient anymore with .backward() function."

- change w font for "Note that we can't reset our w using inplace operation"
